core/tauri/src/api/notification.rs

```diff
@@ -72,6 +74,36 @@ impl Notification {
     self
   }
 
+  /// Sets the notification sound. Silent by default.
+  ///
+  /// ## Platform-specific
+  ///
+  /// Each OS has a different sound name so you will need to conditionally specify an appropriate sound
+  /// based on the OS in use, for a list of sounds see:
+  /// - **Linux**: can be one of the sounds listed in <https://0pointer.de/public/sound-naming-spec.html>
+  /// - **Windows**: can be one of the sounds listed in <https://learn.microsoft.com/en-us/uwp/schemas/tiles/toastschema/element-audio>
+  ///   but without the prefix, for example, if `ms-winsoundevent:Notification.Default` you would use `Default` and
+  ///   if `ms-winsoundevent:Notification.Looping.Alarm2`, you would use `Alarm2`.
+  ///   Windows 7 is not supproted, if a sound is provided, it will play the default sound, otherwise it will be silent.
+  /// - **macOS**: you can specify the name of the sound you'd like to play when the notification is shown
+  /// or use `NSUserNotificationDefaultSoundName` to play the default sound.
```

Comment:
need someone to test this bit

Reply:
@amrbashir actually only the sounds from the Library/Sounds paths are supported: https://github.com/h4llow3En/mac-notification-sys/blob/7ab3864527698472224671093e581d4d1880851f/src/notification.rs#LL339C54-L339C54

Reply:
damn, that's really bad, should we make a PR there to allow for a default sound? I was thinking about adding a method to use default sound on all platforms

Reply:
I think it actually needs to use https://developer.apple.com/documentation/usernotifications/unnotificationsound/1649030-default and it's probably easier to manage that on our end.